src/Microsoft.Extensions.Primitives/ChangeTokenHelper.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Primitives
+{
+    /// <summary>
+    /// Propagates notifications that a change has occured.
+    /// </summary>
+    public static class ChangeTokenHelper
```

Comment:
Ok, will update with ChangeToken instead, I liked that name too, but it almost doesn't sound like a static helper class with that name.